internal/ignition/ignition.go

```diff
@@ -140,6 +159,13 @@ func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte)
 		}
 	}
 
+	if g.cluster.Operators != nil && g.cluster.Operators.Lso != nil && *g.cluster.Operators.Lso.Enabled {
```

Comment:
not tests for ignition?

Reply:
Maybe move that logic inside `generateLsoManifests` so that you could add test to it more easily?

Reply:
Added the tests!